src/fuel-vm/index.md

```diff
@@ -45,12 +45,12 @@ Of the 64 registers (6-bit register address space), the first `16` are reserved:
 | `0x04` | `$ssp`   | stack start pointer | Memory address of bottom of current writable stack area.                         |
 | `0x05` | `$sp`    | stack pointer       | Memory address on top of current writable stack area (points to free memory).    |
 | `0x06` | `$fp`    | frame pointer       | Memory address of beginning of current call frame.                               |
-| `0x07` | `$hp`    | heap pointer        | Memory address below the current bottom of the heap (points to used/oob memory). |
+| `0x07` | `$hp`    | heap pointer        | Memory address below the current bottom of the heap (points to used/OOB memory). |
 | `0x08` | `$err`   | error               | Error codes for particular operations.                                           |
 | `0x09` | `$ggas`  | global gas          | Remaining gas globally.                                                          |
 | `0x0A` | `$cgas`  | context gas         | Remaining gas in the context.                                                    |
 | `0x0B` | `$bal`   | balance             | Received balance for this context.                                               |
-| `0x0C` | `$is`    | instrs start        | Pointer to the start of the currently-executing code.                            |
+| `0x0C` | `$is`    | INSTRs start        | Pointer to the start of the currently-executing code.                            |
```

Comment:
nit: This feels wrong. Instrs is just a short form of Instructions that fits here. Maybe `instrs. start` would be fine?

Reply:
Just write it out if the abbreviation feels unclear